src/scene/vcFlythrough.cpp

```diff
@@ -31,19 +31,25 @@ vcFlythrough::vcFlythrough(vcProject *pProject, udProjectNode *pNode, vcState *p
   m_timePosition = 0.0;
   m_timeLength = 0.0;
   m_exportPath[0] = '\0';
-  m_selectedResolutionIndex = (int)udLengthOf(vcScreenshotResolutions) - 1;
-  m_selectedExportFPSIndex = 3;
-  m_selectedExportFormatIndex = 0;
   memset(&m_exportInfo, 0, sizeof(m_exportInfo));
 
   m_exportInfo.currentFrame = 0;
-  m_exportInfo.frameDelta = (1.0 / (double)vcFlythroughExportFPS[m_selectedExportFPSIndex]);
 
   OnNodeUpdate(pProgramState);
 }
 
 void vcFlythrough::OnNodeUpdate(vcState *pProgramState)
 {
+  const char *pExportPath = nullptr;
+  udProjectNode_GetMetadataString(m_pNode, "exportPath", &pExportPath, "");
+  udStrcpy(m_exportPath, udLengthOf(m_exportPath), pExportPath);
+
+  udProjectNode_GetMetadataInt(m_pNode, "selectedResolutionIndex", &m_selectedResolutionIndex, (int32_t)udLengthOf(vcScreenshotResolutions) - 1);
+  udProjectNode_GetMetadataInt(m_pNode, "selectedExportFPSIndex", &m_selectedExportFPSIndex, 3);
+  udProjectNode_GetMetadataInt(m_pNode, "selectedExportFormatIndex", &m_selectedExportFormatIndex, 0);
```

Comment:
These don't need to be saved- they are transient and once they are in a modal they will just be selected as you export.